docs/Homebrew-and-Python.md

```diff
@@ -73,6 +73,8 @@ Some formulae provide Python bindings.
 
 These should be installed via `pip install <package>`. To discover, you can use <https://pypi.org/search>.
 
+Starting with Python 3.12, we highly recommend you to use a separate virtualenv for this (see the section about PEP688 below).
```

Comment:
Is this "highly recommend" or "must" given I think `pip install` is blocked now?